lib/active_record/virtual_attributes/virtual_arel.rb

```diff
@@ -14,6 +14,30 @@ class Arel::Nodes::Grouping
     end
 
     module VirtualArel
+      class ATProxy < Arel::Table
```

Comment:
Just for my knowledge is `ATProxy` a Rails thing (that you are trying to override from Rails) or is that a construct specifically for this gem.?

Reply:
it is specific for this Gem. ArelTableProxy - suggest a different name if you want

Reply:
changed the name and added a comment.